src/components/dashboard/OutOfGasError.js

```diff
@@ -110,35 +116,37 @@ const getStylesFromProps = ({ theme }) => ({
     flexGrow: 1,
     flexShrink: 0,
     justifyContent: 'space-evenly',
-    paddingTop: getDesignRelativeHeight(33),
+    paddingTop: getDesignRelativeHeight(theme.sizes.default),
     borderRadius: 5,
   },
   mainContainer: {
     paddingBottom: 0,
     paddingTop: 0,
     marginBottom: 0,
     flex: 1,
+    marginHorizontal: getDesignRelativeWidth(0),
   },
   mainSection: {
     padding: 0,
     marginBottom: 0,
+    marginHorizontal: getDesignRelativeWidth(0),
+    paddingHorizontal: getDesignRelativeWidth(12),
   },
   image: {
-    height: getDesignRelativeHeight(146),
+    height: getDesignRelativeHeight(180),
```

Comment:
There's a second parameter to this, max=true/false, currently we dont increase design sizes only decrease for smaller screen, lets experiment with increasing it for larger devices by using the second parameter.
please send some screens on larger screen using second parameter not to limit size

Reply:
I've changed it because of new image. new image has different width/height ratio...

Reply:
```
image: {
    height: getDesignRelativeHeight(180),
  },
```
![image](https://user-images.githubusercontent.com/36693645/65412224-76b8e480-ddf7-11e9-989f-407be82c7e66.png)

```
image: {
    height: getDesignRelativeHeight(225, true),
  },
```

![image](https://user-images.githubusercontent.com/36693645/65412255-8b957800-ddf7-11e9-97c0-3962bfda4d52.png)

@sirpy @LiavGut 
I think first one is better
should I set true and different size only on large device ?